cookbook/form/data_transformers.rst

```diff
@@ -16,24 +16,24 @@ to render the form, and then back into a ``DateTime`` object on submit.
     When a form field has the ``inherit_data`` option set, Data Transformers
     won't be applied to that field.
 
-Simple Example: Sanitizing HTML on User Input
----------------------------------------------
```

Comment:
Can you please add a label to be BC with the old headline? This should look like this:

```rst
.. _simple-example-sanitizing-html-on-user-input:
```